packages/core/src/components/tabs/tabs.scss

```diff
@@ -0,0 +1,183 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/units";
+
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/variables/scaffolding";
+@use "~dso-toolkit/src/variables/typography";
+@use "~dso-toolkit/src/components/anchor";
+
+@use "~dso-toolkit/src/components/tabs/tabs.variables" as css-tabs-variables;
+
+@include utilities.box-sizing();
+
```

Comment:
Volgens mij is het niet de bedoeling dat we de css 2 x uitschrijven; we stoppen alles in een mixin die we 2 x oproepen

Reply:
Dat gaf Eric al aan, maar ik denk dat Eric gelijk heeft dat we eerst de opzet van de componenten helder hebben alvorens een SCSS refactor te doen. Wellicht dat de implementaties wel uit elkaar gaan lopen waardoor er minder SCSS gedeeld kan worden dan we denken.